arbitrum-docs/node-running/how-tos/data-availability-committee/partials/_2.1-das-configuration-options-interfaces.mdx

```diff
@@ -0,0 +1,13 @@
+There are two main interfaces that can be enabled in a DAS: an **RPC interface** to store data in the DAS, intended to be used only by the AnyTrust sequencer; and a **REST interface** that supports only GET operations and is intended for public use.
```

Comment:
I don't know if we're bothering with the Sequencer/Batch Poster distinction here. They can be configured to run on the same instance of Nitro or separate instances.

Reply:
I've only used Sequencer for DAS deployers and Batch poster for DAC chain owners. They are in different guides aimed at different audiences, but thought that those would be the terms that each audience are most familiar with.